frontend/public/components/dashboard/grid.tsx

```diff
@@ -3,6 +3,7 @@ import { Grid, GridItem } from '@patternfly/react-core';
 
 import { useRefWidth } from '../utils/ref-width-hook';
 
+export const MEDIA_QUERY_SM = 576;
```

Comment:
Maybe PF-React also provides such media size breakpoints, so we can reuse them?

Reply:
I couldnt find those breakpoints anywhere in PF :/

Reply:
They are in the `@patternfly/react-tokens` package.

Reply:
@TheRealJon thanks, im using `global_breakpoints` from PF in latest update